editor/editor/mode/text.js

```diff
@@ -1,11 +1,20 @@
+/**
+ * External dependencies
+ */
+import Textarea from 'react-textarea-autosize';
```

Comment:
I suspect this caused the `react` dependency to start being included in the build again? We might need to add an alias to the Webpack configuration to prevent dependencies from trying to use React directly.